prime-router/src/main/kotlin/azure/LookupTableFunctions.kt

```diff
@@ -119,20 +154,24 @@ class LookupTableFunctions(
         @BindingName("tableName") tableName: String,
         @BindingName("tableVersion") tableVersion: Int
     ): HttpResponseMessage {
-        return getOktaAuthenticator().checkAccess(request) {
-            try {
-                val tableInfo = lookupTableAccess.fetchVersionInfo(tableName, tableVersion)
-                if (tableInfo == null)
-                    HttpUtilities.notFoundResponse(
-                        request,
-                        "Table $tableName with version $tableVersion does not exist."
-                    )
-                else
-                    HttpUtilities.okResponse(request, mapper.writeValueAsString(tableInfo))
-            } catch (e: Exception) {
-                logger.error("Unable to fetch lookup table with version $tableVersion", e)
-                HttpUtilities.internalErrorResponse(request)
-            }
+        // Do authentication
+        val claims = AuthenticatedClaims.authenticate(request)
+            ?: return HttpUtilities.unauthorizedResponse(request, authenticationFailure)
+        // No authorization check!   Anyone authenticated in ReportStream is allowed to access this function.
+        logger.info("User ${claims.userName} is authorized for endpoint ${request.uri}")
```

Comment:
Remove or change to debug

Reply:
Keeping auth related logging as 'info' so it remains in the logs.